src/models/v2/channel.ts

```diff
@@ -21,6 +21,7 @@ import type { ServersInterface } from "../servers";
 import type { ServerInterface } from "../server";
 
 import type { v2 } from "../../spec-types";
+import { OperationObject } from "spec-types/v2";
```

Comment:
You have imported types from `v2` version. Please use `v2.OperationObject` in places where you operate on it :)

```suggestion
```

Reply:
Fixed!